### PR TITLE
CLOUD-660: render commit hash

### PR DESCRIPTION
**Pull request by svenwltr**
> https://jira.rebuy.de/browse/CLOUD-660

### CODE REVIEW

File: git/git.go [hunk 1]
```diff
@@ -37,6 +39,31 @@ func New(directory string) (*Git, error) {
 
 }
 
+func (g *Git) CommitID() (string, error) {
+	cmd := exec.Command(g.GitPath, "rev-parse", "--short", "HEAD")
```
Comment (svenwltr) [inline git/git.go]: This is the API for calling git. The branch management should be done a layer above.

Comment (bjoernhaeuser) [inline git/git.go]: I assumed that

**Comment by svenwltr** [comment]
Now we have a value of `gitCommitID` in the templates. We can use it for the images.

@rebuy-de/prp-kubernetes-deployment Please review.

File: git/git.go [hunk 2]
```diff
@@ -37,6 +39,31 @@ func New(directory string) (*Git, error) {
...
+	raw, err := cmd.CombinedOutput()
+	if err != nil {
+		return "", err
+	}
+
+	id := strings.ToLower(strings.TrimSpace(string(raw)))
+
+	matched, err := regexp.MatchString("[0-9a-f]{7}", id)
+	if err != nil {
+		return "", err
```
Comment (bjoernhaeuser) [inline git/git.go]: shouldnt that ne `nil, err`?

Comment (svenwltr) [inline git/git.go]: A string cannot be `nil`. I would have to set the type to `*string`, which isn't very neat.

**Pull request merged**